kani-driver/src/assess/metadata.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::fs::File;
+use std::io::BufWriter;
+use std::path::Path;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+use super::table_builder::TableBuilder;
+use super::table_failure_reasons::FailureReasonsTableRow;
+use super::table_promising_tests::PromisingTestsTableRow;
+use super::table_unsupported_features::UnsupportedFeaturesTableRow;
+use super::AssessArgs;
+
+/// The structure of `.kani-assess-metadata.json` files, which are emitted for each crate.
+/// This is not a stable interface.
+#[derive(Deserialize)]
+pub struct AssessMetadata {
+    pub unsupported_features: Vec<UnsupportedFeaturesTableRow>,
+    pub failure_reasons: Vec<FailureReasonsTableRow>,
```

Comment:
Is this failure to compile? To run tests?

Reply:
I've added doc comments to the `*TableRow` types and the new version of this table